continuous_integration/environment-3.6.yaml

```diff
@@ -3,13 +3,13 @@ channels:
   - conda-forge
 dependencies:
   # required dependencies
-  - python=3.6.*
-  - numpy=1.15.*
-  - pandas=0.25.*
+  - python=3.6
+  - numpy=1.15
+  - pandas=0.25
   # test dependencies
   - pytest
   - pytest-xdist
-  - moto>=1.3.14
```

Comment:
Why remove this minimum version pinning? I have the same question for other packages like `fsspec`

Reply:
min version pinning is important in setup.py and conda recipe. In a requirements.yml file, the only cases where it matters are

1. when another package in the environment suddenly starts being pinned against a max version - not likely as these are all fairly old versions; if it ever happens unit tests will fail anyway
2. when a lower-priority channel defines the version you need (a lower version of a higher priority channel wins) - we have only one channel, and anyway this is only a problem with very fresh releases